applications/MedApplication/custom_io/med_model_part_io.cpp

```diff
@@ -498,13 +514,14 @@ void MedModelPartIO::ReadModelPart(ModelPart& rThisModelPart)
     const int dimension = mpFileHandler->GetDimension();
 
     // read family info => Map from family number to group names aka SubModelPart names
-    const auto groups_by_fam = GetGroupsByFamily(
+    auto groups_by_fam = GetGroupsByFamily(
         mpFileHandler->GetFileHandle(),
         mpFileHandler->GetMeshName());
 
     // create SubModelPart hierarchy
-    for (const auto& r_map : groups_by_fam) {
-        for (const auto& r_smp_name : r_map.second) {
+    for (auto& r_map : groups_by_fam) {
+        for (auto& r_smp_name : r_map.second) {
+            RemovePadding(r_smp_name);
```

Comment:
I dont think this is still necessary, since the group names are already cleaned in L349 (see above)